topics:
  path:
    tf_topic: ["/tf"]
    tf_namespace: [""]
  odometry:
    odom_topic: ["/base/odometry_controller/odometry"]
  vision:
    vision_topic: ["/torso_cam3d_right/rgb/image_raw", "/torso_cam3d_left/rgb/image_raw"]
  laser_scan:
    laser_topic: ["/scan_right", "/scan_left"]
